chainercv/transforms/image/scale.py

```diff
@@ -21,9 +28,14 @@ def scale(img, size):
     if (W <= H and W == size) or (H <= W and H == size):
         return img
```

Comment:
If `fit_short` is `False`, this `return` statement causes an undesired behaviour.